apps/spruce/src/pages/adminSettings/Tabs.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import useScrollToAnchor from "hooks/useScrollToAnchor";
+import { GeneralTab } from "./tabs/GeneralTab/GeneralTab";
+
+export const AdminSettingsTabs = () => {
+  useScrollToAnchor();
+
+  return (
+    <Container>
```

Comment:
I think this is okay for now but will need to change to actual react router routes when the components for event-logs and restart-tasks are introduced. This can be done in a separate ticket.

Reply:
I would recommend creating React Router routes sooner rather than later, preferably soon after [DEVPROD-17282](https://jira.mongodb.org/browse/DEVPROD-17282) is complete